pages/[id].js

```diff
@@ -15,10 +18,33 @@ const StyledLink = styled(Link)`
 `;
 
 export default function DetailsPage({ favoriteIDs, onToggleFavorite }) {
+  const router = useRouter();
+  const query = router.query;
+  console.log(query);
+  const [showToastMessage, setShowToastMessage] = useState(true);
+
+  useEffect(() => {
+    let timeoutId;
+    if (query.isnew !== undefined) {
```

Comment:
```suggestion
    if (query.isnew) {
```

Reply:
We've implemented it this way, as the router push was only to `?isnew` without any value behind the key. To implement your suggested change, we therefore now push to route `?isnew=true` to make it work.